linux/LinuxProcess.c

```diff
@@ -404,10 +406,15 @@ static int LinuxProcess_compareByKey(const Process* v1, const Process* v2, Proce
 
 const ProcessClass LinuxProcess_class = {
    .super = {
-      .extends = Class(Process),
-      .display = Process_display,
-      .delete = Process_delete,
-      .compare = Process_compare
+      .super = {
+         .extends = Class(Process),
+         .display = Process_display,
+         .delete = Process_delete,
+         .compare = Process_compare
+      },
+      .isVisible = Process_isVisible,
+      .matchesFilter = Process_matchesFilter,
+      .compareByParent = Process_compareByParent,
    },
    .writeField = LinuxProcess_writeField,
    .compareByKey = LinuxProcess_compareByKey
```

Comment:
```suggestion
   .compareByKey = LinuxProcess_compareByKey,
```

Reply:
Hmm, nak (for now at least) - I'd prefer it your suggested way, but it'll be inconsistent with so many other htop class definitions then.  I think a strong case could be made though - a single cleanup commit to do this would be better (separate PR to this one though).